tests/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{

```

Comment:
you can use the `extend` functionality of tsconfig to not duplicate properties here. See https://www.typescriptlang.org/tsconfig#extends

Reply:
Yes it is an options, however I would like to keep test related tsconfig separately and writtent explicitly, in case there will be some change in the main file it should not affect all tests.